corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -77,7 +77,7 @@ def update(self, rows):
 
         # Setup
         rows = get_unicode_dicts(rows)
-        self._setup_translation_nodes()
+        self._update_or_create_translation_elements()
```

Comment:
Evan was in favor of element because it was more specific than node, since everything in an xml doc is a node. Agreed that it is generally a losing battle due to the lack of convention.

Reply:
That sounds like I'm blaming Evan for that decision which I don't mean to 😅 I think element > node > el, but because of the lack of convention, the difference isn't important in the grand scheme of things.

Reply:
Ok actually Jenny pointed out a gap in my knowledge, which is that I don't know that there is actually a meaningful difference between element and node. @nospame are you aware of elements being a specific xml convention that makes them "more specific" than nodes?

Reply:
My pretty surface-level knowledge is that "node" is specific to XML, but basically describes anything selectable -- attributes, elements, even the document itself. In this context it seemed like node was the superset and that "element" would be more specific.

Reply:
Attributes aren't nodes. I had thought element and node were exactly equivalent but found [this](https://docs.oracle.com/en/cloud/saas/marketing/responsys-develop/RPL/XMLTerminology.htm) today saying the document doesn't count as an element. So element is a tiny bit more specific.

Reply:
I had looked at [this](https://www.w3schools.com/xml/dom_nodes.asp) saying attributes are a type of node. In any case, I think we're in agreement that element is _slightly_ more specific in this case.

Reply:
Huh, maybe I'm wrong. I see docs saying both, and also some mention of "attribute nodes" 🤷‍♀️